src/keybindings.c

```diff
@@ -1827,11 +1827,11 @@ static void goto_matching_brace(GeanyDocument *doc)
 
 static gboolean cb_func_clipboard_action(guint key_id)
 {
-	GeanyDocument *doc = document_get_current();
+	GtkWidget *focusw = gtk_window_get_focus(GTK_WINDOW(main_widgets.window));
 
-	if (doc == NULL)
+	if (!IS_SCINTILLA(focusw)) {
```

Comment:
@b4n I believe this is because the signal for the find or search and replace dialog is not captured there. The file-browser pane is on the other hand (I assume because it gets focus for the main_window) and fails.